sunpy/net/fido_factory.py

```diff
@@ -423,6 +417,16 @@ def _make_query_to_client(self, *query):
         # UnifiedResponse object.
         return results
 
+    def __repr__(self) -> str:
```

Comment:
Make this the `__str__` and then have the repr be object repr + str again?

Reply:
Fixed.